### PR TITLE
Add few test to travis and appveyor.

### PR DESCRIPTION
**Pull request by lighta**
Updating our script of constant integration to try the login and char-serv as run-once also.
Add some final test system in it. (only travis atm). to try if all server could interconnect themself.

* **Addressed Issue(s)**: 

* **Server Mode**: 

* **Description of Pull Request**:

### CODE REVIEW

**Comment by hurtsky** [comment]
just a request sir @lighta can you add support for gitlab-ci also.

**Comment by Lemongrass3110** [comment]
@hurtsky https://gitlab.com/gitlab-org/gitlab-ce/issues/38091
If this thing gets done by gitlab it makes sense, otherwise we have to mirror all pushes to a copy on gitlab and build an integration for gitlab ci's result for github to display.

**Comment by hurtsky** [comment]
it's okay @Lemongrass3110 , i just give out some suggestion.. because i think there are some of rathena user using gitlab as it can be setup as private repo. i also did try make my own gitlab-ci , just it wont terminate normally like travis and appveyor. thank you.

File: .travis.yml [hunk 1]
```diff
@@ -36,7 +36,10 @@ script:
   - ./configure $CONFIGURE_FLAGS
   - make clean
   - make server
+  - ./login-server --run-once
+  - ./char-server --run-once
   - ./map-server --run-once
+  - ./athena-start start && sleep 60 && ./athena-start stop
```
Comment (aleos89) [inline .travis.yml]: Is there a reason for this addition after the servers have already ran?

**Pull request merged**